app/assets/javascripts/octobox.js

```diff
@@ -479,19 +539,25 @@ var Octobox = (function() {
   return {
     moveCursorToClickedRow: moveCursorToClickedRow,
     checkAll: checkAll,
-    mute: mute,
+    muteThread: muteThread,
+    muteSelected: muteSelected,
     markReadSelected: markReadSelected,
-    archive: archive,
-    unarchive: unarchive,
+    archiveSelected: archiveSelected,
+    unarchiveSelected: unarchiveSelected,
     toggleSelectAll: toggleSelectAll,
     sync: sync,
     markRowCurrent: markRowCurrent,
+    closeThread: closeThread,
```

Comment:
Would be good to have a keyboard shortcut for closing the thread, perhaps `esc`?

Reply:
👍